robot/Cumulus/tests/browser/recurring_donations/create_a_fixed_length_recurring_donation_multiply_by.robot

```diff
@@ -0,0 +1,85 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Create Fixed Length Recurring Donation Multiply By
+    
+    #Create two Contacts in the Same Household
+    ${ns} =  Get NPSP Namespace Prefix
+    &{contact} =                 API Create Contact           Email=jjoseph@robot.com
+    Store Session Record         Account                      &{contact}[AccountId]
+    &{contact2} =                API Create Contact           AccountId=&{contact}[AccountId]
+    
+    #Create a Fixed Length Recurring Donation
+    Go To Record Home            &{contact}[Id]
+    &{recurringdonation} =       API Create Recurring Donation  npe03__Contact__c=&{contact}[Id]
+    ...                          Name=Julian Recurring Donation
+    ...                          npe03__Amount__c=10
+    ...                          npe03__Installments__c=50
+    ...                          npe03__Schedule_Type__c=Multiply By
+    ...                          npe03__Installment_Period__c=Weekly
+    Go To Record Home            &{recurringdonation}[Id]
+
+    #Find 1st Opportunity for Recurring Donation and Close It
+    @{opportunity1} =            Salesforce Query             Opportunity
+    ...                          select=Id
+    ...                          npe03__Recurring_Donation__c=&{recurringdonation}[Id]
+    ...                          ${ns}Recurring_Donation_Installment_Name__c=(1 of 50)
```

Comment:
I see that you are using this piece of code at least 8 other places. Is this something you want to convert into a keyword ? Or do you want to leave it as-is for better readability ? It's your call, just wanted to suggest it.

Reply:
@Safiyab Thanks yeah I think that makes sense long term however right now each query is a little bit different due to how installment names are done (depends on if it's an open or closed RD) and I have a feeling this will change significantly for RD2 so I don't think it would be worth cluttering NPSP.robot with something that will likely change and wouldn't make this test a whole lot cleaner.